Neuro/NeuroPlugin.cs

```diff
@@ -1,208 +1,45 @@
-        Debug.Log(frameString);
-        recorder.Frames.Add(frame);
-    }
-
-    public bool MovePlayer(ref Vector2 direction)
-    {
-        moveDirection = direction;
-        if (currentPath.Length > 0 && pathIndex != -1)
-        {
-            Vector2 nextWaypoint = currentPath[pathIndex];
-
-            while (Vector2.Distance(PlayerControl.LocalPlayer.GetTruePosition(), nextWaypoint) < 0.75f)
-            {
-                pathIndex++;
-                if (pathIndex > currentPath.Length - 1)
-                {
-                    pathIndex = currentPath.Length - 1;
-                    nextWaypoint = currentPath[pathIndex];
-                    break;
-                }
-
-                nextWaypoint = currentPath[pathIndex];
-            }
-
-            directionToNearestTask = (nextWaypoint - (Vector2)PlayerControl.LocalPlayer.GetTruePosition()).normalized;
-
-
-            LineRenderer renderer = arrow;
-            renderer.SetPosition(0, PlayerControl.LocalPlayer.GetTruePosition());
-            renderer.SetPosition(1, PlayerControl.LocalPlayer.GetTruePosition() + directionToNearestTask);
-            renderer.widthMultiplier = 0.1f;
-            renderer.positionCount = 2;
-            renderer.startColor = Color.red;
-        }
-        else
-        {
-            directionToNearestTask = Vector2.zero;
-        }
-
-        return true;
-    }
-
-    public void MeetingBegin()
-    {
-        Debug.Log("NEURO: MEETING CALLED");
-        vision.ReportFindings();
-    }
-
-    public void MeetingEnd()
-    {
-        Debug.Log("NEURO: MEETING IS FINISHED");
-        vision.MeetingEnd();
-    }
-    public IEnumerator EvaluatePath(NormalPlayerTask initial)
-    {
-        currentPath = pathfinding.FindPath(PlayerControl.LocalPlayer.transform.position, initial.Locations[0]);
-        pathIndex = 0;
-
-        while (true)
-        {
-            yield return new WaitForSeconds(1);
-
-            PlayerTask task = PlayerControl.LocalPlayer.myTasks[0];
-
-            PlayerTask nextTask = null;
-            if (task.IsComplete)
-            {
-                Debug.Log("Task is complete");
-                foreach (PlayerTask t in PlayerControl.LocalPlayer.myTasks)
-                {
-                    if (!t.IsComplete && t.HasLocation)
-                    {
-                        nextTask = t;
-                        Debug.Log(nextTask.name);
-                        break;
-                    }
-                }
-            }
-            else
-            {
-                nextTask = task;
-            }
-            if (nextTask != null)
-            {
-                Debug.Log("Next task isn't null");
-                currentPath = pathfinding.FindPath(PlayerControl.LocalPlayer.transform.position, nextTask.Locations[0]);
-                pathIndex = 0;
-
-                //pathfinding.DrawPath(currentPath);
-            }
-        }
+        Harmony.CreateAndPatchAll(Assembly.GetExecutingAssembly(), Id);
```

Comment:
While I dislike `CreateAndPatchAll` in this case it's functionally the same, so it's fine I guess, but then what's the reason for changing it?

Reply:
it's shorter